lib/bemenu.h

```diff
@@ -190,6 +190,7 @@ enum bm_key {
     BM_KEY_SHIFT_TAB,
     BM_KEY_ESCAPE,
     BM_KEY_RETURN,
+    BM_KEY_KP_RETURN,
```

Comment:
No need for this. Map the numpad return to BM_KEY_RETURN in wayland and x11 renderer.

Reply:
The keys here are only for unique actions. Since _RETURN and _KP_RETURN do the same action, we don't need separate enum for it.

Reply:
Thanks!